.github/workflows/publish-paradedb.yml

```diff
@@ -90,9 +90,12 @@ jobs:
           images: paradedb/paradedb
           tags: |
             type=raw,value=${{ matrix.pg_version }}-${{ steps.version.outputs.tag }}
-            type=raw,value=${{ steps.version.outputs.tag }},enable=${{ matrix.pg_version == 16 }}
-            type=raw,value=${{ steps.version.outputs.version }},enable=${{ matrix.pg_version == 16 }}
             type=raw,value=latest,enable=${{ matrix.pg_version == 16 }}
+            type=raw,value=${{ steps.version.outputs.tag }},enable=${{ matrix.pg_version == 16 }}
+            type=semver,pattern={{version}},value=${{ steps.version.outputs.tag }},enable=${{ matrix.pg_version == 16 }}
+            type=raw,value=latest-pg${{ matrix.pg_version }}
+            type=raw,value=${{ steps.version.outputs.tag }}-pg${{ matrix.pg_version }}
+            type=semver,pattern={{version}},value=${{ steps.version.outputs.tag }},suffix=-pg${{ matrix.pg_version }}
```

Comment:
What are the two `semver` types doing here? They seem to be duplicates of the `raw` types with the exact same version strings. Do they do anything different?

From what you have here, it is looking to me like the following would be the most concise way to express it? Am I missing something?

```yaml
            type=raw,value=${{ matrix.pg_version }}-${{ steps.version.outputs.tag }}
            type=raw,value=${{ steps.version.outputs.tag }}-pg${{ matrix.pg_version }}
            type=raw,value=latest-pg${{ matrix.pg_version }}
            type=raw,value=latest,enable=${{ matrix.pg_version == 16 }}
            type=raw,value=${{ steps.version.outputs.tag }},enable=${{ matrix.pg_version == 16 }}
```

Reply:
`type=raw,value=${{ steps.version.outputs.tag }}-pg${{ matrix.pg_version }}`
outputs the tag as it triggered the gh workflow (eg: `v0.11.0`)

`type=semver,pattern={{version}},value=${{ steps.version.outputs.tag }},suffix=-pg${{ matrix.pg_version }}`
outputs a semver parse of the tag that triggered the gh workflow (eg: `0.11.0`)

Considering that we are no-longer tracking `major` and `major+minor` I could simply revert back to your original implementation using the `steps.version.outputs.version` workflow variable. But it could be useful if in the future you want to readd those two tags. Let me know

Reply:
Updated back to raw for simplicity for now.